dart/lib/src/sentry_client.dart

```diff
@@ -56,6 +65,14 @@ class SentryClient {
       _options.logger(SentryLevel.debug, 'No scope is defined');
     }
 
+    if (stackTrace != null) {
+      event = event.copyWith(
+        stackTrace: SentryStackTrace(
+          frames: _stackTraceFactory.getStackFrames(stackTrace),
+        ),
+      );
+    }
```

Comment:
this looks like being done twice, here and https://github.com/getsentry/sentry-dart/pull/160/files#r521245971

Reply:
✅